alerta-covid/app/views/areas/_area.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! area, :id, :institution_id, :name, :created_at, :updated_at
+json.url area_url(area, format: :json)
```

Comment:
if you remove the respond to Json in web controller, this file can also be removed

Reply:
the json response is needed for the module's api